feat: documentation for oracle swaps (#649)
chainflip-io/chainflip-docs: pages/swapping/swapping-basics.mdx

6. The swap deposit reaches the witness threshold and is executed. Each swap consumes some range orders before the pool price matches the next best limit order.
7. The Chainflip Validator network sends the swapped ETH funds to the user's ETH wallet address at the end.

## Slippage protection

Chainflip allows 2 forms of slippage protection, Minimum price, and maximum oracle slippage.

> This allows Users to grantee a satisfactory result for the swap or you get a refund.

Reviewer: The protocol guarantees the result, not the users (unless you meant something else?). How about:
```suggestion
This makes sure the user gets a satisfactory result for the swap or they get a refund.
```

The protocol also requires a refund address and a retry duration. 
A retry duration is the time window (measured in blocks) during which a swap must be executed before the deposit is refunded to the refund address.
If one of the provided forms of protection are not satisfied for the specified retry duration, the deposited assets will be refunded to the specified refund address on the source chain.
These protections are enforced on the AMM level and do not take deposit fees and broadcast fees into account,
but the Network and broker fees will effect these protections and must be taken into account when setting them.
If a deposit is refunded, the Chainflip protocol deducts a deposit fee and a broadcast fee from the refund amount to pay for the transactions sent by the protocol.
If the deposit was boosted, the boost fee is not refunded.

### Minimum price

If a minimum accepted price is provided, the Chainflip protocol will only swap the deposited assets if the available liquidity leads to a price that is equal or better than the specified minimum price.

### Maximum oracle price slippage

Aka Live price protection. In units of **basis points**.

> This form of protection is not available to all assets,

Reviewer: I would swap order of this line with the next. Better to say what something is before talking about its limitations.
